src/clientmanager.cpp

```diff
@@ -209,7 +209,9 @@ Client* ClientManager::manage_client(Window win, bool visible_already, bool forc
     changes = Root::get()->rules()->evaluateRules(client, stdio, changes);
     if (!changes.manage || force_unmanage) {
         // map it... just to be sure
```

Comment:
Should we update the comment?

Reply:
Yes, I've tried to say something more meaningful.

Reply:
Awesome, thank you!